web/style/transvision.css

```diff
@@ -507,6 +508,20 @@ fieldset {
     background: url(/img/swap_icon_horiz_18x18.png) center no-repeat transparent;
 }
 
+#search_field_container {
+    position: relative;
+    display: inline;
+}
+
+#clear {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 40px;
+    height: 22px;
```

Comment:
When resized to smaller viewports the X is visibly off-center vertically. We can use 20px for all sizes, or add rules to viewports and keep 22px for the big one.
![schermata 2015-06-17 alle 14 36 03](https://cloud.githubusercontent.com/assets/3644868/8206968/36b15776-14fe-11e5-92e5-44c60e395c44.png)